pkg/ccl/backupccl/show_test.go

```diff
@@ -573,34 +575,34 @@ func TestShowBackupTenants(t *testing.T) {
 	tenant10.Exec(t, `CREATE DATABASE foo; CREATE TABLE foo.bar(i int primary key); INSERT INTO foo.bar VALUES (110), (210)`)
 	beforeTS := systemDB.QueryStr(t, `SELECT now()::timestamp::string`)[0][0]
 
-	systemDB.Exec(t, fmt.Sprintf(`BACKUP TENANT 10 TO 'nodelocal://1/t10' AS OF SYSTEM TIME '%s'`, beforeTS))
+	systemDB.Exec(t, fmt.Sprintf(`BACKUP TENANT 10 INTO 'nodelocal://1/t10' AS OF SYSTEM TIME '%s'`, beforeTS))
```

Comment:
I'd be inclined to leave the tests for the old syntax in place as long as it's supported? Though we could copy them and test with the new syntax as well.

Reply:
Good point. Done.